tests/test_content.py

```diff
@@ -327,7 +327,7 @@ def test_subpackage_metadata_preserved():
     err2 = ErrorBundle()
     err2.detected_type = PACKAGE_EXTENSION
 
-    submain.test_package(err1, xpi1, 'jetpack-1.8-outdated.xpi')
+    submain.test_package(err1, xpi1, 'jetpack-1.16-outdated.xpi')
```

Comment:
It was. 1.17 is the current version, that makes 1.16 an outdated one :)
I cannot use a version older than 1.16 as the tests rely on the hashes
being in the allow list.

But for various reasons, we don't allow 1.15 or older, that's why I removed
all those hashes from the allow list.

On Friday, 4 December 2015, Mathieu Agopian <notifications@github.com>
wrote:

> In tests/test_content.py
> <https://github.com/mozilla/amo-validator/pull/363#discussion_r46722442>:
>
> > @@ -327,7 +327,7 @@ def test_subpackage_metadata_preserved():
> >      err2 = ErrorBundle()
> >      err2.detected_type = PACKAGE_EXTENSION
> >
> > -    submain.test_package(err1, xpi1, 'jetpack-1.8-outdated.xpi')
> > +    submain.test_package(err1, xpi1, 'jetpack-1.16-outdated.xpi')
>
> Given the name of the XPI, wasn't it there in the tests on purpose to test
> outdated XPIs?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/amo-validator/pull/363/files#r46722442>.
>